.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
```

Comment:
Is that an overall Sienna workflow change (I am in favor). Do we need developer instructions? We could also solve this in a way that does not require a Python installation.

Reply:
We are rolling it out slowly and see how it goes. We can add this to the instructions, for me using home-brew was a very easy thing to do.